src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -366,6 +379,31 @@ private void validateName(final String name, final EventTypeBase eventType) thro
         }
     }
 
+    private void validateAuthorization(final EventTypeBase eventType) throws InvalidEventTypeException,
```

Comment:
It looks like we have view object in a service layer. Maybe we move `EventTypeAuthorization` to view package and prepare model object for authorization? From other hand it does not make much sense, because we pass EventTypeBase view object in the calling method.